### PR TITLE
test: enable GH actions

### PR DESCRIPTION
**Pull request by decentral1se**
@staltz 

e5b0770023f9ab07f340326e13604be8caa4b993 fixes the tests :woman_shrugging:

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2022 Henry Bubert
```
Comment (staltz) [inline .github/workflows/go.yml]: Could you change the copyright to yourself? :)

**Comment by staltz** [comment]
I don't know why CI is not running here.

File: .github/workflows/go.yml (hunk 1 above)
Comment (decentral1se) [reply .github/workflows/go.yml]: Done :upside_down_face:

**Comment by staltz** [comment]
I merged hoping that CI will run in the main branch.

**Pull request merged**